docs/source/release_notes/release-notes.rst

```diff
@@ -42,6 +42,39 @@ develop
     *    - Type
          - Change
 
+    *    -
+         -
+
+
+======
+0.45.0
+======
+
+19 June 2017
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Type
+         - Change
+
+    *    - |devbreak| |improved|
+         - Upgraded all usages of http-remoting to remoting2.
+           Previously, depending on the use case, AtlasDB would use http-remoting, remoting1 and remoting2.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1999>`__)
+
+    *    - |devbreak| |improved|
+         - AtlasDB now ships with OpenFeign 9.5.0 and OkHttp 3.6.0.
```

Comment:
Again, point out that devs may have inherited our versions and need ot check that they still work with the new ones.